src/wp-includes/class-wp-theme-json.php

```diff
@@ -1260,6 +1260,7 @@ public function get_settings() {
 	 *     @type string $scope                   Makes sure all style are scoped to a given selector
 	 *     @type string $root_selector           Overwrites and forces a given selector to be used on the root node
 	 *     @type bool   $skip_root_layout_styles Omits root layout styles from the generated stylesheet. Default false.
+	 *     @type bool   $block_style_variations  Includes styles for block style variations in the generated stylesheet. Default false.
```

Comment:
Include this new type in line. 1250

Reply:
Done: https://github.com/WordPress/wordpress-develop/pull/6827/commits/7a72f569f820b706d0bae5837d043e002eb07933